dockerfiles/Dockerfile.dev

```diff
@@ -3,19 +3,25 @@ FROM    golang:1.8.3-alpine
 
 RUN     apk add -U git make bash coreutils
 
-ARG     VNDR_COMMIT=9909bb2b8a0b7ea464527b376dc50389c90df587
-RUN     git clone https://github.com/LK4D4/vndr.git "/go/src/github.com/LK4D4/vndr" && \
-        cd "/go/src/github.com/LK4D4/vndr" && \
-        git checkout -q "$VNDR_COMMIT" && \
+ARG     VNDR_SHA=9909bb2b8a0b7ea464527b376dc50389c90df587
+RUN     go get github.com/LK4D4/vndr && \
+        cd /go/src/github.com/LK4D4/vndr && \
+        git checkout -q "$VNDR_SHA" && \
         go build -v -o /usr/bin/vndr . && \
         rm -rf /go/src/* /go/pkg/* /go/bin/*
 
+ARG     BINDATA_SHA=a0ff2567cfb70903282db057e799fd826784d41d
 RUN     go get github.com/jteeuwen/go-bindata/go-bindata && \
-        cp /go/bin/go-bindata /usr/bin && \
+        cd /go/src/github.com/jteeuwen/go-bindata/go-bindata && \
+        git checkout -q "$BINDATA_SHA" && \
+        go build -v -o /usr/bin/go-bindata . && \
```

Comment:
I wonder if it would be a little more idiomatic to change `$PATH` and `go install` into `/go/bin`. I don't think it really matters though - installing the tools into `/usr/bin` of a container image is reasonable.